<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlurkView</class>
 <widget class="QWidget" name="PlurkView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="qualifierCombo"/>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="contentEdit"/>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="plurkBtn">
     <property name="text">
      <string>Plurk</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QScrollArea" name="plurkListScroll">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>386</width>
        <height>126</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="plurkListWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>91</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>All plurks</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>My plurks</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButton_3">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Private</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButton_4">
     <property name="minimumSize">
      <size>
       <width>91</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Responded</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QPushButton" name="refreshBtn">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
